<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextBrowser" name="ReceivetextBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>361</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
background-color: rgb(188, 188, 141);
font: 75 8pt &quot;Adobe 宋体 Std L&quot;;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="inputbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>361</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>350</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>你的ID：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>好友列表</string>
    </property>
   </widget>
   <widget class="QLabel" name="user_id_number">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>user_id</string>
    </property>
   </widget>
   <widget class="QListWidget" name="list_friend_number">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>40</y>
      <width>101</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>目标ID：</string>
    </property>
   </widget>
   <widget class="QLabel" name="target_id_number">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="add_friend">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>310</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>加好友</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_group">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>310</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>加群</string>
    </property>
   </widget>
   <widget class="QListWidget" name="list_group_number">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>180</y>
      <width>101</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>160</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>群列表</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDataAcq">
    <property name="title">
     <string>Mychat</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAuthor_GGZ">
    <property name="title">
     <string>author:GGZ</string>
    </property>
   </widget>
   <addaction name="menuDataAcq"/>
   <addaction name="menuAuthor_GGZ"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
